lib/std/c/darwin.zig

```diff
@@ -435,6 +435,10 @@ pub const VM_REGION_BASIC_INFO_COUNT: mach_msg_type_number_t = @sizeOf(vm_region
 pub const VM_REGION_EXTENDED_INFO_COUNT: mach_msg_type_number_t = @sizeOf(vm_region_extended_info) / @sizeOf(natural_t);
 pub const VM_REGION_TOP_INFO_COUNT: mach_msg_type_number_t = @sizeOf(vm_region_top_info) / @sizeOf(natural_t);
 
+pub fn VM_MAKE_TAG(tag: i32) i32 {
```

Comment:
Is the `tag` guaranteed to be `i32` always?

Reply:
nope should be `u8 `but returning `u32` then.